packages/calypso-e2e/src/index.ts

```diff
@@ -2,3 +2,5 @@
  * Internal dependencies
  */
 export * as BrowserManager from './browser-manager';
+export * as MediaHelper from './media-helper';
```

Comment:
IMO, we could keep these helpers internal since they aren't used outside of the package. (unless they are :P)

Reply:
Good point, I thought I _had to_ export in order for the package to work. Will make it internal.

Reply:
you should be able to import the media helper file directly from the screenshots hooks file

Reply:
Removed the export in the current revision.